src/snowflake/cli/plugins/object/stage/manager.py

```diff
@@ -54,20 +66,24 @@ def quote_stage_name(name: str) -> str:
 
         return standard_name
 
+    @staticmethod
+    def remove_stage_prefix(stage_path: str) -> str:
+        if stage_path.startswith("@"):
```

Comment:
Should we transform the stage path with `get_standard_stage_prefix` first or are we sure we won't get `snow://` here?

Reply:
I'm not sure about `snow://` so I added it

Reply:
We have this `startswith` in two places. Should we adjust the `get_standard_stage_prefix` to return always the `@...` version? In this way we fill have one place where we handle the `snow://` scenario